client/blocks/stats-navigation/index.js

```diff
@@ -94,6 +108,12 @@ export default connect( ( state, { siteId } ) => {
 			siteId
 		),
 		isStore: isSiteStore( state, siteId ),
+		stickers: siteId ? getBlogStickers( state, siteId ) : false,
+		isWordAds:
+			getSiteOption( state, siteId, 'wordads' ) &&
+			stickers &&
```

Comment:
I understand `stickers` is not defined here, but I'm creating that property above on line 111. How can I reuse that variable here to show this feature based on whether a blog sticker is present?

Reply:
You are inside a function, so you can create a local variable and use it:
```js
const stickers = getBlogStickers( state, siteId );
return {
  stickers,
  isWordAds: stickers && somethingElse && someOtherThing
}
```

You can avoid some unneeded conditionals, too:
- `getBlogStickers` can cope with null `siteId` and will return `null`. Almost all Calypso selectors are like this
- `lodash.includes` can take `null` as the first parameter and treats it as "empty collection". All Lodash methods that work with arrays/collections (`map`, `find`, ...) are like this.